app/main.py

```diff
@@ -1,12 +1,70 @@
-from fastapi import FastAPI, Request
+from database import models, schemas
+from database.database import engine, SessionLocal
+from fastapi import Depends, FastAPI, Request
 from fastapi.templating import Jinja2Templates
+from sqlalchemy.orm import Session
+import crud
 
+models.Base.metadata.create_all(bind=engine)
 
 app = FastAPI()
 
 templates = Jinja2Templates(directory="templates")
```

Comment:
you should update your branch to match current commit, some things have changed.